### ISSUE TITLE
Syntax for if custom SEO image field

### ISSUE THREAD

**Issue by alanski66** [nystudio107/craft-seomatic#1078]
### Question
I have two entry types ina channel that have different image fields.
entry.image
entry.oldimage

Id like to set the channel SEO content settings so that I can say if entry this field use this image or if entry has this field use oldimage

I thought something along these lines might work:

`{{ seomatic.helper.socialTransform(entry.image[0], "base", 1, "crop") ???seomatic.helper.socialTransform(entry.oldNewsImage[0], "base", 1, "crop") }}`

but alas it did not. I wasn't sure if i had to use socialTransform or could use existing transform. Had a peek in other posts and docs but couldn't quite see if this was legitimate use or not.

tia

**Comment by khalwat** [comment]
Maybe try something like:

```twig
{{ seomatic.helper.socialTransform(entry.image[0] ??? entry.oldNewsImage[0], "base", 1, "crop") }}
```

?

**Comment by alanski66** [comment]
Thanks for the quick response. 
I tried that but the seo preview still remains blank.
I also tried just {{ seomatic.helper.socialTransform(entry.image[0], "base", 1, "crop") }} as this appears to be whats assigned when actually selecting the field and then changing it to custom - that is intended behaviour right to show the underlying twig code?
It works if i just set the field but its a shame to lose the goodness for the older articles, or have to manually override each one.
